### PR TITLE
Corrected a minor typo in 'Change Password' page

### PR DESCRIPTION
**Pull request by wadamT**
## Pull Request Description

Formatted 'next line' correctly in Account Settings> Change Password page.

## Issue Being Fixed

Screenshot_20240614-040829_Thunder

### CODE REVIEW

File: lib/l10n/app_en.arb [hunk 1]
```diff
@@ -2633,4 +2633,4 @@
   "@yes": {
     "description": "A positive status"
   }
-}
\ No newline at end of file
+}
```
Comment (hjiangsu) [inline lib/l10n/app_en.arb:2636]: Make sure to remove the newline here so that it passes the CI!

Comment (wadamT) [reply lib/l10n/app_en.arb]: Done.

Comment (micahmo) [reply lib/l10n/app_en.arb]: @wadamT Your PR now includes a new file, `lib/app_en.arb`. Once you delete that, this should be good to go!

**Review by micahmo — APPROVED** [review]

File: lib/l10n/app_en.arb (hunk 1 above)
Comment (wadamT) [reply lib/l10n/app_en.arb]: @micahmo Done.

**Pull request merged**